cli/src/spend_utils.rs

```diff
@@ -96,7 +105,7 @@ where
         )?;
         Ok((message, spend))
     } else {
-        let from_balance = rpc_client
+        let mut from_balance = rpc_client
             .get_balance_with_commitment(from_pubkey, commitment)?
             .value;
         let from_rent_exempt_minimum = if amount == SpendAmount::RentExempt {
```

Comment:
Here

Reply:
I made change to get the `from_rent_exempt_minimum` with `SpendAmount::Available` case.
Then I subtract from the balance 👍

Reply:
Is this issue still open intentionally? I'm looking for ways to pitch in, but don't want to duplicate efforts.